order/supplier/receive/simple-order-event.md

```diff
@@ -0,0 +1,249 @@
+---
+description: How to receive a simple order sent by the buyer.
+---
+
+# Receive an order
```

Comment:
Note: somehow this is not the header that is rendered on this page..

Reply:
I see this
![image](https://github.com/tradecloud/tradecloud-docs-api-v2/assets/1092388/e86fee01-5519-40d9-bb45-ba389fbcf0b0)

Reply:
![Screenshot 2024-03-05 at 14 49 34](https://github.com/tradecloud/tradecloud-docs-api-v2/assets/46396634/e5034a29-d838-481d-b8a8-af452f000424)

Are we looking at the same page? I'm looking at `processes/order/supplier/receive`. It seems your screenshot if of `processes/order/supplier/README.md`?

Reply:
The Gitbook uses the title from the glossary

Reply:
Alright, I've pushed a small change to make it consistent with the summary. Just to avoid confusion